src/components/CallToAction/index.tsx

```diff
@@ -75,12 +77,8 @@ export const CallToAction = ({
         </>
     )
 
-    return href ? (
-        <a href={href} className={classList}>
-            {innerHtml}
-        </a>
-    ) : to ? (
-        <Link to={to} className={classList}>
+    return href || to ? (
```

Comment:
I think this should be okay @smallbrownbike as our custom Link component (changed from Gatsby link) handles both internal and external destinations... but please let me know if you think otherwise!

Reply:
We should actually just delete this file and rename CallToAction.js to CallToAction.tsx. It already handles this logic! Happy to help with types here if needed 😄